gen_cert.py

```diff
@@ -1841,8 +1832,24 @@ def fontlist_with_style(a_style):
         yOffset = minYOffset + ((max_height - height) / 2)
         paragraph.drawOn(PAGE, GUTTER_WIDTH - (name_style.fontSize / 12), yOffset)
 
+        successfully_completed = ''
+        grade_interstitial = ''
+        disclaimer_text = ''
+        verify_text = ''
+
+        if self.locale in settings.DEFAULT_TRANSLATIONS:
+            successfully_completed = settings.DEFAULT_TRANSLATIONS[self.locale]['success_text']
+            grade_interstitial = settings.DEFAULT_TRANSLATIONS[self.locale]['grade_interstitial']
+            disclaimer_text = settings.DEFAULT_TRANSLATIONS[self.locale]['disclaimer_text']
+            verify_text = settings.DEFAULT_TRANSLATIONS[self.locale]['verify_text']
+
+        if self.course_translations and self.locale in self.course_translations:
```

Comment:
Since we default `self.course_translations` to an empty dict (`self.course_translations = cert_data.get('translations', {})`), we can remove the first half of this `if` clause, as it will never be `None`.